opflexagent/utils/port_managers/async_port_manager.py

```diff
@@ -101,7 +101,7 @@ def apply_config(self):
             with excutils.save_and_reraise_exception():
                 # The upper layers will trigger the resync
                 LOG.error("Configuration failed on port manager: %s",
-                          e.message)
+                          str(e))
```

Comment:
Are you sure the str() is needed here? I would have thought that the format API handled this for you?

Reply:
Yes, str() prints the Exception message for both py2 and py3, and Exception.message was deprecated & removed in python3.